conftest.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import absolute_import, unicode_literals
+
+
+def pytest_addoption(parser):
+    parser.addoption("--xonsh-prompt", action="store_true", help="Run (do not skip) xonsh prompt behavior tests")
```

Comment:
why shouldn't this be a flag?

Reply:
Clarity/explicitness, mainly.

I'm not tied to it, though -- what would you prefer?

Reply:
I meant can't we always run this when xonosh is installed? Why have a flag to enable/disable it?

Reply:
The prompt handling in xonsh is mostly sited *within `xonsh`*, sniffing `$VIRTUAL_ENV`, `$VIRTUAL_ENV_DISABLE_PROMPT`, and `$VIRTUAL_ENV_PROMPT`. It would be very easy for a change to be made in xonsh that starts breaking virtualenv's CI builds. 

Thus, I disabled the xonsh tests by default, to keep the CI clean and to try to minimize the codebase cross-coupling, while still making it easy to test xonsh for consistency in a local dev environment.

Reply:
I'm not comfortable with this. We either support or don't support something. If we support xonosh we should always run it to make sure it works. The solution could be to raise an issue in xonosh but we should always fun it.

Reply:
:+1:

Totally cool by me to strip this out, and just always run the xonsh tests. I actually figured that's how it would end up. But, I wanted to make sure you were specifically aware of the potential for this kind of cross-coupled breakage.

Reply:
Yeah let's rip it out, 👌 I'm aware, all shells potentially breaks us (fish did recently with version 3), but we'll know as soon as the CI runs.

Reply:
Done in 2dcd1e2